tree/20_cfe_basics/ncf_lib.cf

```diff
@@ -672,3 +672,38 @@ body edit_defaults ncf_empty_select(select)
         max_file_size             => "25M";
         edit_backup               => "timestamp";
 }
+
+
+# Add parameter in key->values line, where -> is key_value_separator
```

Comment:
Couldn't this be replaced by `edit_line ncf_add_parameter_in_list_with_enclosing_chars` with empty separators?